client/src/.env.development.local.sample

```diff
@@ -0,0 +1 @@
+REACT_APP_STRIPE_PUBLISHABLE_KEY=pk_test_51KJhK0ANCC3EPL26kStEMdsyYC7mI0DmPelXzz3OkZK1tMnpPgy07ZFFFwdbbNxKaJMvfnoxu1m7wGar5qIxFMe600GH1LLEQr
```

Comment:
So, even though this is a public key - I still would advise against putting it in the sample client env far. But good thinking on adding that file here!

Reply:
Same as above.